plugins/chat/assets/javascripts/discourse/components/chat-message.hbs

```diff
@@ -30,6 +31,7 @@
       @emojiReactions={{this.emojiReactions}}
       @secondaryButtons={{this.secondaryButtons}}
       @messageActions={{this.messageActions}}
+      @messageActionsHandler={{@messageActionsHandler}}
```

Comment:
I had to do `@messageActionsHandler` here because the ember linter scolded me for doing `this.args.messageActionsHandler`